### ISSUE TITLE
Additional Docker Env Variables

### ISSUE THREAD

**Issue by robertminer1** [seatable/seatable#121]
Hi!  If possible, it would be great to be able to use specific environment variable references for redis & memcached in docker compose.  Ex:

```
REDIS_HOST: container-name
MEMCACHED_HOST: container-name
```
Also, is there a reason why the script `/shared/seatable/scripts/seatable.sh start` isn't included as a line in the entrypoint script?  Just wondering!

Thank you!

**Comment by freeplant** [comment]
> Also, is there a reason why the script /shared/seatable/scripts/seatable.sh start isn't included as a line in the entrypoint script? Just wondering!

Because there are some manual steps needed before running seatable yet. So we don't add it in the entrypoint script.

**Comment by SkywalkerSpace** [comment]
> If possible, it would be great to be able to use specific environment variable references for redis & memcached in docker compose
Would you mean to add these two environment variables to the yml file？We didn’t add these because they were not used in the docker container.

**Comment by robertminer1** [comment]
Got 

> > Also, is there a reason why the script /shared/seatable/scripts/seatable.sh start isn't included as a line in the entrypoint script? Just wondering!
> 
> Because there are some manual steps needed before running seatable yet, we don't add it in the entrypoint script.

Got it, thanks.

**Comment by robertminer1** [comment]
> > If possible, it would be great to be able to use specific environment variable references for redis & memcached in docker compose
> 
> Would you mean to add these two environment variables to the yml file？We didn’t add these because they were not used in the docker container.

Yes, that's what I was asking for.  I would like to be able to set the redis server and memcached host dynamically.
